examples/stm32/xively_demo_stm32f4xx_nucleo_wifi/SW4STM32/STM32F401RE-Nucleo/xively_demo_stm32f4xx-nucleo-wifi/.cproject

```diff
@@ -111,6 +112,127 @@
 						</toolChain>
 					</folderInfo>
 					<fileInfo id="fr.ac6.managedbuild.config.gnu.cross.exe.debug.1474710539.110071304" name="LinkerScript.ld" rcbsApplicability="disable" resourcePath="LinkerScript.ld" toolsToInvoke=""/>
+					<sourceEntries>
+						<entry excluding="Drivers/BSP/Components/lis2dh12|Drivers/BSP/Components/h3lis331dl|Drivers/BSP/X_NUCLEO_IKS01A2" flags="VALUE_WORKSPACE_PATH|RESOLVED" kind="sourcePath" name=""/>
+					</sourceEntries>
+				</configuration>
+			</storageModule>
+			<storageModule moduleId="org.eclipse.cdt.core.externalSettings"/>
+		</cconfiguration>
+		<cconfiguration id="fr.ac6.managedbuild.config.gnu.cross.exe.debug.1474710539.895893754">
+			<storageModule buildSystemId="org.eclipse.cdt.managedbuilder.core.configurationDataProvider" id="fr.ac6.managedbuild.config.gnu.cross.exe.debug.1474710539.895893754" moduleId="org.eclipse.cdt.core.settings" name="IKS01A2_Debug">
+				<macros>
+					<stringMacro name="XI_STMCUBE_PATH" type="VALUE_PATH_ANY" value="/Users/palantir/Work/Xively/stm32f4xx/STM32CubeExpansion_WIFI1_V2.1.1"/>
+					<stringMacro name="XI_CLIENT_PATH" type="VALUE_PATH_ANY" value="/Users/palantir/Work/Xively/stm32f4xx/xively-client-c"/>
+					<stringMacro name="XI_MEMS_STMCUBE_PATH" type="VALUE_PATH_ANY" value="/Users/palantir/Work/Xively/stm32f4xx/STM32CubeExpansion_MEMS1_V3.0.0"/>
```

Comment:
I've tried using the Linked Resources from the Build Variables and vice versa, but I keep getting build errors. I'd be surprised if it's really not possible, but I can't see how to do it.

Leaving this as it is for now, and we can update it when  we find out what's going on. Maybe the CC3200 FW upgrade project will give us some insight into why/how it works on the TI demos.